docs/examples/viz_network_animated_billboard.py

```diff
@@ -0,0 +1,282 @@
+# The network initially is a 1D ring of `vertices_count` vertices
+# connected with their `ws_neighs` neighbors. Then the edges are
+# randomly rewired with probability `ws_rewiring`.
+
+vertices_count = 10000
+ws_neighs = 3
+ws_rewiring = 0.01
+view_size = 100
+
+edges_list = []
+for i in range(vertices_count):
+    for k in range(ws_neighs):
+        if(random.random() >= ws_rewiring):
+            edges_list.append((i, (i+k+1) % vertices_count))
+        else:
+            while(True):
+                random_from = random.randint(0, vertices_count)
+                random_to = random.randint(0, vertices_count)
+                if(random_from != random_to):
+                    break
+            edges_list.append((random_from, random_to))
+edges = np.ascontiguousarray(edges_list, dtype=np.uint64)
+
+
+###############################################################################
+# Defining visual properties, such as scale and colors of nodes
+
+labels = None
+
+positions = view_size * \
+    np.random.random((vertices_count, 3)) - view_size / 2.0
+
+positions = np.ascontiguousarray(positions, dtype=np.float32)
+colors = np.array(cmap.cm.inferno(
+    np.arange(0, vertices_count)/(vertices_count-1)))
+radii = 1 + np.random.rand(len(positions))
+
+edges_colors = []
+for source, target in edges:
+    edges_colors.append(np.array([colors[source], colors[target]]))
+
+edges_colors = np.average(np.array(edges_colors), axis=1)
+
+
+###############################################################################
+# Our data preparation is ready, it is time to visualize them all. We start to
+# build 2 actors that represent our data : nodes_actor for the nodes and
+# lines_actor for the edges.
+
+n_points = colors.shape[0]
+np.random.seed(42)
+centers = np.zeros(positions.shape)
+radius = np.ones(n_points)
+
+lines_actor = actor.line(np.zeros((len(edges), 2, 3)),
+                         colors=edges_colors, lod=False,
+                         fake_tube=False, linewidth=3,
+                         opacity=0.1
+                         )
+
+
+###############################################################################
+# We use a billboard actor to render the spheres for nodes
+# so that a shader is used to draw a sphere into each billboard.
+
+billboard_sphere_dec = shader_load("billboard_spheres_dec.frag")
+billboard_sphere_impl = shader_load("billboard_spheres_impl.frag")
+
+nodes_actor = actor.billboard(centers,
+                              colors=colors,
+                              scales=1.0,
+                              fs_dec=billboard_sphere_dec,
+                              fs_impl=billboard_sphere_impl
+                              )
+
+###############################################################################
+# Preparing editable geometry for the nodes
+
+vtk_centers_geo = vtk_array_from_actor(nodes_actor, array_name="center")
+centers_geo = vtknp.vtk_to_numpy(vtk_centers_geo)
```

Comment:
this step is not needed if you use `array_from_actor` function

Reply:
Fixed: Removed all vtk_array_* calls